### ISSUE TITLE
Unable to resolve dependencies with Rails 5

### ISSUE THREAD

**Issue by jznadams** [chargify/chargify_api_ares#142]
I've previously been using the Chargify gem by including the following in my gemfile:

`gem "chargify_api_ares"
`

 I wanted to start testing my application against Rails 5 today, but when I changed the Rails gem dependency I noticed that the Chargify gem downgraded itself significantly when I ran `bundle update`:

`gem "rails", "5.0.0.1"`

`Installing chargify_api_ares 0.4.4 (was 1.4.7)`

When I go back and explicitly specify the version of the Chargify gem I want to use, bundler is unable to resolve the discrepancies:

`gem "chargify_api_ares", '1.4.7'
`

```
    chargify_api_ares (= 1.4.7) was resolved to 1.4.7, which depends on
      activeresource (>= 3.2.16) was resolved to 3.2.16, which depends on
        activesupport (= 3.2.16)
```

Any ideas?

**Comment by santervo** [comment]
Chargify depends on activeresource gem which is released as separate gem only up to version 4.1 which does not work with rails 5. You can solve the issue by depending to rails 5 version of activeresource via github in your Gemfile:

```
gem 'activeresource', github: 'rails/activeresource', branch: 'master'
gem "chargify_api_ares", '~> 1.4.7'
```

**Comment by jznadams** [comment]
It looks like this has _magically_ resolved itself with the latest versions of the chargify_api_ares/rails so I will close this for now.
